api/bootstrap.proto

```diff
@@ -74,6 +74,10 @@ message ClusterManager {
   // Optional configuration used to bind newly established upstream connections.
   // This may be overridden on a per-cluster basis by upstream_bind_config in the cds_config.
   BindConfig upstream_bind_config = 3;
+
+  // A management server endpoint to stream load stats to, as per
```

Comment:
I thought the load stats go back via EDS itself? Or did we change that. I can't remember.